liferay-frontend/liferay-portal
# LPS-137760 Remove stale *EditorConfigContributor classes in Blogs

Context: https://liferay.slack.com/archives/CLD39UKM5/p1631122808008200

These classes were leftovers of a replacement of liferay-editor tag with native elements(a.k.a inputs) in some parts of blogs.

### How to test it?

Deploy and navigate around blogs things including widget and see any error being thrown.

/cc @boton @adolfopa


It also includes the removal of stale editor classes in forms in https://github.com/liferay-frontend/liferay-portal/pull/1465/commits/9e980ca75543e5b85d5cb25290a46cc1021d5855. As agreed in https://liferay.slack.com/archives/CL8R04Z7X/p1631646432102300?thread_ts=1631645043.100700&cid=CL8R04Z7X
